src/driver.ts

```diff
@@ -51,23 +66,35 @@ export interface Device {
  * or `null` if a matching device cannot be found connected to your computer.
  */
 export const findDevice = (): Device | null => {
-  const matchingDevice = HID.devices().find(
-    (device) =>
-      device.vendorId === VENDOR_ID &&
-      PRODUCT_IDS.includes(device.productId) &&
-      device.usagePage === USAGE_PAGE,
-  );
+  const matchingDevice = HID.devices().find((device) => isLitraDevice(device));
 
   if (matchingDevice) {
-    return {
-      type: getDeviceTypeByProductId(matchingDevice.productId),
-      hid: new HID.HID(matchingDevice.path as string),
-    };
+    return HIDDeviceToDevice(matchingDevice);
```

Comment:
See above for context

```suggestion
    return hidDeviceToDevice(matchingDevice);
```